src/Security/Authentication/Certificate/samples/Certificate.Optional.Sample/Program.cs

```diff
@@ -10,8 +12,8 @@ namespace Certificate.Optional.Sample;
 
 public class Program
 {
-    public const string HostWithoutCert = "127.0.0.1";
-    public const string HostWithCert = "127.0.0.2";
+    public const string HostWithoutCert = "example.com";
+    public const string HostWithCert = "cert.example.com";
```

Comment:
These values didn't need to change, and it actually breaks running the sample unless someone adds HOST file entries for them. Please revert, and keep that note in the readme.